README.md

```diff
@@ -109,6 +110,14 @@ Or as an ES6 module, from an .mjs file:
 import * as openpgp from 'openpgp';
 ```
 
+#### Deno (experimental)
+
+Import as an ES6 module, using /dist/openpgp.mjs.
+
+```js
+import * as openpgp from "./openpgpgjs/dist/openpgp.mjs";
```

Comment:
Something like this?

Reply:
Yep, seems reasonable for now :+1: One small typo (and quote style):
```suggestion
import * as openpgp from './openpgpjs/dist/openpgp.mjs';
```